### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by SaharFarhangi** [neuropoly/axondeepseg#794] [Help needed]
I opened Miniconda on my desktop and type activate ads_venv but it shows an error:
(base) C:\>activate ads-venv

EnvironmentNameNotFound: Could not find conda environment: ads-venv
You can list all discoverable environments with `conda info --envs`.

**Comment by mathieuboudreau** [comment]
Hi Sahar! Did you do the step just before, https://axondeepseg.readthedocs.io/en/latest/documentation.html#install-axondeepseg, i.e. after already have opened (i.e. `cd`-ing) the AxonDeepSeg directory in your terminal:


<img width="584" alt="Screenshot 2024-03-11 at 7 37 18 PM" src="https://github.com/axondeepseg/axondeepseg/assets/1421029/6f5cc6f4-9933-4a83-b2dc-bb655a120b13">

The "open directory" step in our instructions are just above that, where we talk about cloning (i.e. downloading) the project:

<img width="584" alt="Screenshot 2024-03-11 at 7 38 26 PM" src="https://github.com/axondeepseg/axondeepseg/assets/1421029/d1fc77ba-a81d-4b0d-81ad-b2b3ee1c00b4">

**Comment by Stoyan-I-A** [comment]
I noticed that you wrote `ads-venv` instead of `ads_venv`. Is this intentional?

**Reply by mathieuboudreau** [reply]
Good catch Stoyan!
